native/src/navigation/navigateToLink.ts

```diff
@@ -50,7 +48,7 @@ const navigateToLink = async <T extends RoutesType>(
         url,
       },
     })
-    const { pathname } = new Url(url)
+    const { pathname } = new URL(url)
```

Comment:
Probably makes sense but can you explain why this change?

Reply:
i don't think we need a separate library if its build-in